README.md

```diff
@@ -1,11 +1,57 @@
+
+# Getting Started
+Vulkan-Hpp is part of the LunarG Vulkan SDK since version 1.0.24. Just include ```<vulkan.hpp>``` and you're ready to use the C++ bindings. If you're using a Vulkan version not yet supported by the Vulkan SDK you can find the latest version of the header [here](https://github.com/KhronosGroup/Vulkan-Hpp/blob/master/vulkan/vulkan.hpp).
+
+# Minimum Requirements
+Vulkan-Hpp requires a C++11 capable compiler to compile. The following compilers are known to work:
+* Visual Studio >=2013
+* GCC >= 4.8.2 (earlier version might work, but are untested)
+* Clang >= 3.3
+
+# namespace vk
+To avoid name collisions with the Vulkan C API the C++ bindings reside in the vk namespace. The following rules apply to the new naming
+* All functions, enums and structs have the Vk prefix removed.
+  * ```vkCreateImage``` can be accessed as ```vk::CreateImage```
+  * ```VkImageTiling``` can be accessed as ```vk::ImageTiling```
+  * ```VkImageCreateInfo``` can be accessed as ```vk::ImageCreateInfo```
+* Enums are mapped to scoped enums to provide compile time type safety. The names have been changed to 'e' + CamelCase with the VK_ prefix and type infix removed. In case the enum type is an extension the extension suffix has been removed from the enum values.
+ In all other cases the extension suffix will has not been removed.
+  * ```VK_IMAGETYPE_2D``` is now ```vk::ImageType::e2D```.
+  * ```VK_COLOR_SPACE_SRGB_NONLINEAR_KHR``` is now ```vk::ColorSpaceKHR::eSrgbNonlinear```.
+  *  ```VK_STRUCTURE_TYPE_PRESENT_INFO_KHR``` is now ```vk::StructureType::ePresentInfoKHR```.
+*  Flag bits are handled like scoped enums with the addition that the _BIT suffix has also been removed.
+
+# Handles
+Vulkan-Hpp declares a class for all handles to ensure full type safety and to add support for member functions on handles. A member function has been added to a handle class for each
+ function which accepts the corresponding handle as first parameter. Instead of ```vkBindBufferMemory(device, ...)``` one can write ```device.bindBufferMemory(...)``` or ```vk::bindBufferMemory(device, ...)```.
+
+# C/C++ Interop for Handles
+On 64-bit platforms Vulkan-Hpp supports implicit conversions between C++ Vulkan handles and C Vulkan handles. On 32-bit platforms all non-dispatchable handles are defined as ```uint64_t``` and thus not typesafe. 
+Due to that Vulkan-Hpp does not enable implicit conversion for 32-bit platforms by default and it is recommended to use a static_cast for the conversion like this: ```VkDevice = static_cast<VkDevice>(cppDevice)```
+to prevent converting some arbitrary int to a handle or vice versa by accident. If you're developing your code on a 64-bit platform, but want compile your code for a 32-bit platform without adding the explicit casts
+you can define ```VULKAN_HPP_TYPESAFE_CONVERSION``` to 1 in your build system or before including ```vulkan.hpp```.
```

Comment:
It would be worth clarifying whether setting VULKAN_HPP_TYPESAFE_CONVERSION would affect 64-bit builds at all.